packages/myst-cli/src/process/mdast.ts

```diff
@@ -223,6 +225,14 @@ export async function transformMdast(
   // Combine file-specific citation renderers with project renderers from bib files
   const fileCitationRenderer = combineCitationRenderers(cache, ...rendererFiles);
 
+  const serverSettings = ServerConnection.makeSettings({
+    baseUrl: process.env.JUPYTER_BASE_URL,
+    token: process.env.JUPYTER_TOKEN,
+  });
```

Comment:
I would be curious on the vision here eventually on how to hook this up to the config.